Longest_Increasing_Subsequence/LIS.go

```diff
@@ -0,0 +1,65 @@
+/* Problem Statement :
+To find the length of the Longest Increasing Subsequence of a given sequence, that is all the elements of the subsequence are in increasing sorted order.
```

Comment:
Can you please split this comment across multiple lines. 80 characters per line is standard.